### PR TITLE
Remove dox

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
Based on gtk-rs/gir#1005.

### CODE REVIEW

File: gdk4-wayland/sys/src/lib.rs [hunk 1]
```diff
@@ -134,10 +134,6 @@ impl ::std::fmt::Debug for GdkWaylandToplevel {
 
 #[link(name = "gtk-4")]
 extern "C" {
-
-    //=========================================================================
-    // GdkWaylandDevice
-    //=========================================================================
```
Comment (bilelmoussaoui) [inline gdk4-wayland/sys/src/lib.rs:140]: regen checker would complain about this

Comment (GuillaumeGomez) [reply gdk4-wayland/sys/src/lib.rs]: Just updated my rustfmt and force-pushed. ;)

**Comment by GuillaumeGomez** [comment]
I wonder: is it valid in a toml file to have an empty "category" like `[features]`?

**Comment by GuillaumeGomez** [comment]
@bilelmoussaoui This is ready.

**Comment by bilelmoussaoui** [comment]
Thanks!

**Pull request merged**